api/proto/cloudprovider/kubernetes.proto

```diff
@@ -130,3 +130,8 @@ message KubernetesAccountDockerRegistry {
     // The list of Docker registry namespaces usable as image sources.
     repeated string namespaces = 2;
 }
+
+enum KubernetesProviderVersion {
```

Comment:
Should we just delete this, since by the time Kleat is GA v1 will likely be totally unsupported?

Reply:
Good call, deleted!